### DISCUSSION TITLE
🐝🐝🐝 Could you help me with adding your buzzword that you like to my list

### DISCUSSION THREAD

**Discussion by KKKKKXY** [NAIST-SE/AttractingContributors#828] [2023 Attraction Contributions]
I need help with my buzzword list.

Come to my fork:
https://github.com/KKKKKXY/AttractingContributors_Buzzwords

Add to my README file.

I look forward to your contributions

**Comment by ALORAMFER** [comment]
Can you help me please? haha
https://github.com/ALORAMFER/AttractingContributors-SoccerTeams

**Reply by KKKKKXY** [reply]
OKIEEE!!
